tcms/xmlrpc/tests/test_testcase.py

```diff
@@ -1 +1,47 @@
 # -*- coding: utf-8 -*-
+
+from django import test
+
+from tcms.xmlrpc.api import testcase as XmlrpcTestCase
+from tcms.xmlrpc.tests.utils import make_http_request
+from tcms.xmlrpc.tests.utils import user_should_have_perm
+
+from tcms.tests.factories import TestCaseFactory
+from tcms.tests.factories import UserFactory
+
+__all__ = (
+    'TestNotificationRemoveCC',
+)
+
+
+class TestNotificationRemoveCC(test.TestCase):
+    ''' Tests the XMLRPM testcase.notication_remove_cc method '''
```

Comment:
Oh, many code including those docstrings quoted by  ``'''`` are really old and written several years ago. For now, double quote should be used to quote docstrings. This is the style I've been using in all my Python code. Sorry for confusion.

It would be fine to open an issue to fixed them all at once. Feel free to do that and fix them if you like.